### ISSUE TITLE
Vertex colors?

### ISSUE THREAD

**Issue by insomniacUNDERSCORElemon** [planetis-m/naylib#108]
From what I gather this is an issue with Raylib itself and code exists to read+display vertex colors from 3D models but between the language difference and time (plus my place at square 0 and no other reliable place to ask, particularly specific to Naylib) it is beyond me.

For some background, I generally want my art without textures (vertex color only).

Example:

![planter_low](https://github.com/planetis-m/naylib/assets/4397725/a3841fcf-2ec7-4bf1-890f-00dc3d8d122e)

but in the model display example `yaw_pitch_roll.nim` I swap my model in (~manually scaled up by x100, exported to .glb~ exported to .obj) and get this:

![Screenshot_2024-04-16_19-32-02](https://github.com/planetis-m/naylib/assets/4397725/341c1d25-f774-435c-9c0a-0b637646d271)

EDIT: I just noticed this too for the .glb `Color attribute data format not supported`. Using .obj does not give this message but still does not have color (though the scaling during exporting actually gives a normal size unlike the .glb export). Exporting without a material makes no difference other than making it white rather than gray.

Also my exports do have vertex colors, visible by importing into Godot 3.X or re-importing back into Blender.

I don't want every object using a unique material (this is something to note with Godot and vertex colors), though it would be nice to have the ability to have objects that contain a matte and reflective material (and maybe glass too). Unshaded is not out of the question, 1 material only (+vertex lighting) is a step better.

Note: I am on Kbin, I often seem to have federation issues just talking about Nim (also for the threads I posted w/the art itself and asking questions), I'm not sure if there's somewhere better on the fediverse that might be more conducive for discussion of things like this. Forgive me if this is out-of-place.

**Comment by insomniacUNDERSCORElemon** [comment]
> Sorry I can't answer the question, but you can ask at the raylib discord, probably the feature request channel.

I do not use Discord. 

That and I would expect this to not work out directly when it comes to Nim/Naylib. It would probably be more likely if someone who understands both C and Nim could convert the code others have posted (assuming it can work here).

**Comment by planetis-m** [comment]
I should clarify I am not knowledgeable enough to answer your question. But if you use https://chat.lmsys.org/ (opus is the best one with less rate-limit exceeded errors) you can translate your code to any language. And then maybe create a new raylib issue, if that's what it is or get some help in the nim matrix server.

**Comment by insomniacUNDERSCORElemon** [comment]
@planetis-m I realized that Raylib itself has added more vertex color support since I posted this, but also trying to update Naylib and it seems I was likely using an old version before when I posted this. After installing again, I see via an error that it's still trying to use `naylib-4.6.1-b8dba6d8ec2d85d03665077430f315612b508f4c`.

I installed via nimble and I cannot see a clear place on how to fix this.
